enhancements/hypershift/networking/networking.md

```diff
@@ -104,19 +104,29 @@ egressSelections:
 For components that do not natively support Konnectivity, Hypershift has a [sidecar](https://github.com/openshift/hypershift/tree/main/konnectivity-socks5-proxy)
 that provides a socks5 proxy that can be used by configuring the `HTTP{,S}_PROXY` and `NO_PROXY` env vars. The sidecar
```

Comment:
I assume these proxy variables are configured in the payload you're attaching your sidecontainer to, right? It might we worth mentioning, too.